clients/fides-js/src/lib/consent-types.ts

```diff
@@ -100,6 +100,9 @@ export type FidesOptions = {
 
   // Encodes cookie as base64 on top of the default JSON string
   base64Cookie: boolean;
+
+  // Allows specifying the preferred locale used for translations
+  fidesLocale?: string;
```

Comment:
Could you explain what happens if I use geolocation query param like this when running PC: `http://localhost:3000/?geolocation=us-nc`, but also have `fidesLocale` set up to `fr-CA`? 

I think the experience call will still use `us_nc`, and return notices for that region, but I should see the `fr` lang in the UI if we have that lang configured in translations?

Reply:
Yes, you're correct. The locale is only used for client-side translations and doesn't affect the experience fetch.